src/main/java/org/folio/circulation/domain/notice/NoticeContextUtil.java

```diff
@@ -2,37 +2,66 @@
 
 import org.folio.circulation.domain.Item;
 import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.Request;
 import org.folio.circulation.domain.User;
 import org.joda.time.DateTimeZone;
 
 import io.vertx.core.json.JsonObject;
 
 public class NoticeContextUtil {
 
+  private static final String PATRON = "patron";
+  private static final String ITEM = "item";
+
   private NoticeContextUtil() {
   }
 
-  public static JsonObject createNoticeContextFromLoan(Loan loan) {
-    return createNoticeContextFromLoan(loan, DateTimeZone.UTC);
+  public static JsonObject createLoanNoticeContext(Loan loan) {
+    return createLoanNoticeContext(loan, DateTimeZone.UTC);
   }
 
-  public static JsonObject createNoticeContextFromLoan(Loan loan, DateTimeZone timeZone) {
+  public static JsonObject createLoanNoticeContext(Loan loan, DateTimeZone timeZone) {
```

Comment:
Can't we do something like this to reduce code duplication:
```suggestion
  public static JsonObject createLoanNoticeContext(Loan loan, DateTimeZone timeZone) {
    return createNoticeContextFromItem(loan.getItem(), loan.getUser())
        .put("dueDate", loan.getDueDate().withZone(timeZone).toString());
  }
```

Reply:
Done